src/name_service/delivery/delete.clj

```diff
@@ -0,0 +1,33 @@
+(ns name-service.delivery.delete
+  (:require [clojure.spec.alpha :as s]
+            [name-service.delivery.spec :as spec]
+            [name-service.delivery.utils :as utils]
+            [name-service.delivery.state :as state]
+            [cheshire.core :as json]))
```

Comment:
Please sort imports. Look at https://stuartsierra.com/2016/clojure-how-to-ns.html
```suggestion
  (:require [cheshire.core :as json]
            [clojure.spec.alpha :as s]
            [name-service.delivery.spec :as spec]
            [name-service.delivery.state :as state]
            [name-service.delivery.utils :as utils]))
```